maturity-model.md

```diff
@@ -0,0 +1,175 @@
+spread though. The solution is to provide a maturity model to allow for teams to
+go through a self check and discover patterns and practices that they are not
+yet aware of.
+
+## Problem
+
+When InnerSource adoption in an enterprise starts to increase individual
+mentoring of each project through one evangelist becomes unfeasible. Also more
+and more people are gaining at least a basic understanding of what it means to
+work in an InnerSource project. Looking at all InnerSource projects though the
+depth of understanding for the concept will diverge. Teams may not be aware of
+proven patterns that would help them move to the next level and solve issues and
+pain points that they are dealing with.
+
+## Context
+
+Several teams have started adopting InnerSource practices. The exact level of
+understanding of the practice diverges between teams. The exact problems teams
+run into diverge depending on the context and working environment of each team.
+As a result the definition of what are important best practices in an
+InnerSource project differs depending on each team.
+
+## Forces
+
+Teams sharing InnerSource learnings run into misunderstandings as they are not
+aware of their respective level of InnerSource adoption.
+
+Teams believe that "it's all about migrating to a shared software development
+forge".
+
+Teams are not aware of best practices that would help them solve issues that
+they run into in their daily doing.
+
+## Solution
+
+Ask teams to self assess against the proposed maturity model.
+
+### Transparency
+
+**Plans & Products**
+
+* PP-0: Individuals and teams do not regularly disclose their plans or products to multiple stakeholders. No formal actions exists at the organization.
+* PP-1: Individuals and teams give visibility to their plans or products to multiple stakeholders, before they are started. Shared roadmap.
+* PP-2: Stakeholders can contribute to some plans or products.
+* PP-3: There is an standard process for collective product and plans definition.
+
+
+**Development Process & Tools**
+
+* DP-0:	Each team follows its own development process and tools. They are not defined to share knowledge and artifacts outside development team. Siloed development teams.
+* DP-1: Development teams use shared code repositories, internally. Some teams develop their own CI process, using non corporate or standard CI tools. There is no code review process defined, although some projects teams do it internally.
+* DP-2: The organization sponsors and promotes a shared repository for collective knowledge. Some teams develop their own CI process, using corporate CI tools. There are CI environments. Code review process defined, and used by some projects. Code review is sometimes done by external team members.
+* DP-3: Most teams develop their own CI process, using corporate CI tools. There are CI environments. Code review process defined, and used by some projects. Code review is usually done by external team members.
```

Comment:
I think we can simplify this to the fact that there is a proper collaboration in place. If we reach DP-3, this would mean that code review is already in place, and people and different business units are reviewing each other.